app/views/shared/nav/_nav_links.html.haml

```diff
@@ -14,13 +17,14 @@
               %li= subnav_link.to_html
       - else
         = link.to_html
+  
 
 -# .visible-with-nav is visible > 979px
 = form_tag global_search_path, class: "navbar-search pull-right visible-with-nav" do
   = text_field_tag :search, nil, placeholder: t("global_search.search"), class: "search-query span2"
 
 -# trailing "<" enables use of .nav-collapse .nav:empty in boostrap-responsive-overrides.scss
 %ul.nav.pull-right<
-  = render "shared/nav/manage_facilities"
+  
```

Comment:
Can we use the feature flag to leave this where it was? I think moving it will make sense once some of the other design changes are in (making use and manage pills and the rest tabs), but for now, I think it would confuse people if it just moves (we know why because we know what's coming later, but the users would probably consider it arbitrary).